packages/itwinui-react/src/core/Alert/Alert.tsx

```diff
@@ -37,73 +147,106 @@ export type AlertProps = {
   children: React.ReactNode;
 } & Omit<CommonProps, 'title'>;
 
+export const AlertComponent = React.forwardRef(
+  (props: AlertProps, ref: React.RefObject<HTMLDivElement>) => {
+    const {
+      children,
+      className,
+      type = 'informational',
+      style,
+      isSticky = false,
+      ...rest
+    } = props;
+
+    useTheme();
+
+    return (
+      <div
+        className={cx('iui-alert', className)}
+        data-iui-status={type}
+        data-iui-variant={isSticky ? 'sticky' : undefined}
+        style={style}
+        ref={ref}
+        {...rest}
+      >
+        {children}
+      </div>
+    );
+  },
+) as PolymorphicForwardRefComponent<'div', AlertOwnProps>;
+
 /**
  * A small box to quickly grab user attention and communicate a brief message
  * @example
- * <Alert>This is a basic alert.</Alert>
- * <Alert type='positive'>This is a positive alert.</Alert>
- * <Alert type='warning'>This is a warning alert.</Alert>
- * <Alert type='negative'>This is a negative alert.</Alert>
- * <Alert
- *  type='positive'
- *  clickableText="I am a clickable text"
- *  clickableTextProps={{ href: 'https://www.example.com' }}
- * >
- *   This is a positive alert with a clickable text
+ * <Alert>
+ *  <Alert.Message>This is an alert.</Alert.Message>
+ * </Alert>
+ * <Alert type='informational'>
+ *  <Alert.Icon type='informational' />
+ *  <Alert.Message>This is an informational alert.</Alert.Message>
  * </Alert>
+ * <Alert type='positive'>
+ *  <Alert.Icon>
+ *    <SvgSmileyHappy />
+ *  </Alert.Icon>
+ *  <Alert.Message>
+ *    This is an alert.
+ *    <Alert.ClickableText>This is clickable text.</Alert.ClickableText>
+ *  </Alert.Message>
+ *  <Alert.CloseButton onClose={action('Close!')}>
+ *    <Alert.CloseButtonIcon>
+ *      <SvgCollapse />
+ *    </Alert.CloseButtonIcon>
+ *  </Alert.CloseButton>
  */
-export const Alert = (props: AlertProps) => {
-  const {
-    children,
-    className,
-    type = 'informational',
-    clickableText,
-    clickableTextProps,
-    onClose,
-    style,
-    isSticky = false,
-    ...rest
-  } = props;
+export const Alert = Object.assign(AlertComponent, {
+  /**
+   * 	Alert icon subcomponent
+   */
+  Icon: AlertIcon,
+  /**
+   * 	Alert message subcomponent
+   */
+  Message: AlertMessage,
+  /**
+   * 	Alert clickable text subcomponent for the link you want to provide.
+   */
+  ClickableText: AlertClickableText,
```

Comment:
`Alert.Action` is a better name for this

Reply:
Changed